content/en/docs/guides/certificate_management/certificate-rotation.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Root Certificate Rotation"
+description: "Rotating the root certificate in OSM"
+type: docs
+weight: 10
+---
+
+# Root Certificate Rotation
+
+The Mesh's Root certificate is a long-lived certificate that is used by OSM to issue leaf certificates to services in the mesh thereby enabling mTLS. This document focuses on the steps to rotate a certificate, to learn more about OSM's certificates see [Certificate Provider Options](certificates.md).  
+
+The certificate can be rotated either through a [manual process](#manual-root-certificate-rotation) or with OSM v1.3 we have introduced a preview feature to help [automate rotation](#certificate-rotation-with-meshrootcertificate) and minimize downtime using a new API called the `MeshRootCertificate`.  
```

Comment:
We haven't released v1.3 yet, should the PR keep the `do-not-merge` label until we've released it?
Or we could remove the reference to it for now and add it when we've released?

Reply:
I don't think merging this goes anywhere so it would be safe to merge.  A docs release needs to happen before it is published anywhere.  

I am not sure what the protocol is here, I think I saw 1.2 mentioned like this in other docs but not sure what should happen here

Reply:
As long as we aren't backporting this docs change to v1.2 I think we are good to merge this.